cloud/pkg/cloudhub/servers/httpserver/server.go

```diff
@@ -69,18 +69,22 @@ func StartHTTPServer() {
 // getCA returns the caCertDER
 func getCA(w http.ResponseWriter, r *http.Request) {
 	caCertDER := hubconfig.Config.Ca
-	w.Write(caCertDER)
+	_, _ = w.Write(caCertDER)
 }
 
 //electionHandler returns the status whether the cloudcore is ready
 func electionHandler(w http.ResponseWriter, r *http.Request) {
 	checker := hubconfig.Config.Checker
 	if checker.Check(r) != nil {
 		w.WriteHeader(http.StatusNotFound)
-		w.Write([]byte(fmt.Sprintf("Cloudcore is not ready")))
+		if _, err := w.Write([]byte(fmt.Sprintf("Cloudcore is not ready"))); err != nil {
+			klog.Errorf("failed to write, err: %v", err)
+		}
 	} else {
 		w.WriteHeader(http.StatusOK)
-		w.Write([]byte(fmt.Sprintf("Cloudcore is ready")))
+		if _, err := w.Write([]byte(fmt.Sprintf("Cloudcore is ready"))); err != nil {
+			klog.Errorf("failed to write, err: %v", err)
```

Comment:
better to be more verbose ,  this is helpful for debugging,  but I am good as is.